src/scicookie/{{cookiecutter.project_slug}}/Makefile

```diff
@@ -115,6 +115,9 @@ build:
 	hatch build
 {%- elif cookiecutter.build_system == "maturin" %}
 	maturin build	
+{%- elif cookiecutter.build_system == "pybind11" %}
+	pybind11 build	
```

Comment:
Are you sure this command exists?

Reply:
I think you can use something like `python3 -m build`  https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#packaging-your-project

ensure you have `build` as a dependency in pyproject.toml/build-system